EazFixer.Code/Entrypoint.cs

```diff
@@ -0,0 +1,87 @@
+using System.IO;
+using System.Linq;
+using EazFixer.Code.Other;
+using EazFixer.Code.Processors;
+
+namespace EazFixer.Code {
+    public class Entrypoint {
+        public static int Start(string[] args) {
+            Logger.Initialize("EazFixerLog.md", "[~]");
```

Comment:
Is logging really needed? Only a few lines should be printed out, which could easily be done in the console.